controllers/src/main/scala/cromwell/pipeline/controller/UserController.scala

```diff
@@ -14,32 +14,38 @@ import scala.util.{ Failure, Success }
 class UserController(userService: UserService)(implicit executionContext: ExecutionContext) {
 
   val route: AccessTokenContent => Route = accessToken =>
-    path("users") {
-      concat(
-        get {
-          parameter('email.as[String]) { email =>
-            onComplete(userService.getUsersByEmail(email)) {
-              case Success(r) => complete(r)
-              case Failure(exc) =>
-                complete(StatusCodes.InternalServerError, exc.getMessage)
+    concat(
+      path("users") {
```

Comment:
Let's make it DRY. You don't need concatenation here and can reuse `path("users")` for `put` endpoints
```
path(users) {
  concat (
    get { },
    delete { },
    path(info) { put { } },
    path(password) { put { } }
  )
}
```
or
```
path(users) {
  concat (
    get { },
    delete { },
    put {
      concat(
        path(info) { },
        path(password) { }  
      )
    }
  )
}
```
Which one to choose is up to you.

Reply:
fixed.